linear_algebra/code/CMakeLists.txt

```diff
@@ -3,11 +3,20 @@ project(wg21_linear_algebra VERSION 0.0.1)
 
 option(BUILD_TESTING "flag to build tests" ON)
 
+include(FetchContent)
+FetchContent_Declare(mdspan GIT_REPOSITORY https://github.com/kokkos/mdspan.git)
+FetchContent_GetProperties(mdspan)
+if(NOT mdspan_POPULATED)
+    FetchContent_Populate(mdspan)
+    list(APPEND CMAKE_MODULE_PATH ${mdspan_SOURCE_DIR})
+    add_subdirectory(${mdspan_SOURCE_DIR})
+endif()
+
+
 add_library(wg21_linear_algebra INTERFACE)
 target_include_directories(wg21_linear_algebra
     INTERFACE
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/extsrc/mdspan/include/experimental>
```

Comment:
This is not needed any more, we get this for free simply by depending upon the `std::mdspan` target and transitive includes propagate to users of wg21_linear_algebra.